<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Pictures/icons/mail-account.png</normaloff>../../Pictures/icons/mail-account.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sent mails</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tab_sent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Receiver</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/user.png</normaloff>../../Pictures/icons/user.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Message</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/sent (1).png</normaloff>../../Pictures/icons/sent (1).png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/time-machine (1).png</normaloff>../../Pictures/icons/time-machine (1).png</iconset>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tab_received">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Sender</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/user.png</normaloff>../../Pictures/icons/user.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Message</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/inbox.png</normaloff>../../Pictures/icons/inbox.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/icons/time-machine (1).png</normaloff>../../Pictures/icons/time-machine (1).png</iconset>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Received mails</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_filter_received">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter email to filter letters from a particular user.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Filter by senders...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_admin" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(226, 226, 255, 255), stop:1 rgba(215, 241, 255, 255));</string>
    </property>
    <widget class="QTableWidget" name="tab_users">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>651</width>
       <height>381</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Pictures/icons/employee-card.png</normaloff>../../Pictures/icons/employee-card.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Pictures/icons/circled-envelope.png</normaloff>../../Pictures/icons/circled-envelope.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Place of registration</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Pictures/icons/address.png</normaloff>../../Pictures/icons/address.png</iconset>
      </property>
     </column>
    </widget>
    <widget class="QTextBrowser" name="txt_a_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The address of the last mail of a given user.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>   Address of the last msg sent</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_inactive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Find all user who hasn't sent any message yet.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);</string>
     </property>
     <property name="text">
      <string>Find inactive users</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select option.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Sent to</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Received from</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txt_a_email">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Email adress to get information about.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter login...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_greeting">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionLog_in"/>
    <addaction name="actionSign_up"/>
    <addaction name="actionSign_out"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuNew_message">
    <property name="title">
     <string>Message</string>
    </property>
    <addaction name="actionWrite"/>
    <addaction name="actionDrafts"/>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuNew_message"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLog_in"/>
   <addaction name="actionWrite"/>
  </widget>
  <action name="actionLog_in">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/login-rounded-right (1).png</normaloff>../../Pictures/icons/login-rounded-right (1).png</iconset>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionSign_up">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/add-user-male--v1.png</normaloff>../../Pictures/icons/add-user-male--v1.png</iconset>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/hand-with-pen.png</normaloff>../../Pictures/icons/hand-with-pen.png</iconset>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionSign_out">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/logout-rounded.png</normaloff>../../Pictures/icons/logout-rounded.png</iconset>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionDrafts">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/external-blank-page-user-interface-inkubators-basic-outline-inkubators.png</normaloff>../../Pictures/icons/external-blank-page-user-interface-inkubators-basic-outline-inkubators.png</iconset>
   </property>
   <property name="text">
    <string>Draft</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/icons/pencil.png</normaloff>../../Pictures/icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
